src/doom/g_game.c

```diff
@@ -437,9 +437,9 @@ void G_BuildTiccmd (ticcmd_t* cmd, int maketic)
             joybspeed = MAX_JOY_BUTTONS;
         }
 
-        M_snprintf(playermessage, sizeof(playermessage), "ALWAYS RUN %s%s",
+        M_snprintf(playermessage, sizeof(playermessage), "ALWAYS RUN %s%s%s",
             crstr[CR_GREEN],
-            (joybspeed >= MAX_JOY_BUTTONS) ? "ON" : "OFF");
+            (joybspeed >= MAX_JOY_BUTTONS) ? "ON" : "OFF", crstr[CR_NONE]);
```

Comment:
Hm, doesn't the same happen with the "vertical mouse movement on/off" message?

Reply:
Maybe we should just add a `dp_translation = NULL;` at the very end of `hu_lib.c:HUlib_drawTextLine()`.